### PR TITLE
Format and change alignment of quantity figures on aid package creation table

### LINKED ISSUE #132: Thousand separators for quantity figures
- [ ] Thousand separators for figures to make it easier for admin users to work with medicine quantities
Screen Shot 2022-07-12 at 09 57 19

### PR DESCRIPTION
**Pull request by YaiyaSam**
## Purpose

The purpose of this PR is to fix #132

## Goals

Format the quantities with thousand separators and left align data in the column. 

### Screenshots

image

### CODE REVIEW

**Comment by jayasanka-sack** [comment]
Hi @YaiyaSam , There are some recent changes to the same component. 
https://github.com/LSFLK/MedicinesforLK/pull/162

The provided screenshot is a bit different. Could you please check?

File: admin-portal/src/pages/aidPackage/assignSuppliers/assignSuppliers.tsx [hunk 1]
```diff
@@ -179,9 +179,17 @@ export function AssignSuppliers({
           return (
             <>
               <tr {...row.getRowProps()} className="needs-row">
-                {row.cells.map((cell) => {
-                  return (
-                    <td {...cell.getCellProps()}>{cell.render("Cell")}</td>
+                {row.cells.map((cell, index) => {
+                  return index == 5 || index == 6 ? (
```
Comment (jayasanka-sack) [inline admin-portal/src/pages/aidPackage/assignSuppliers/assignSuppliers.tsx:183]: Column order can be changed if we later decide to display more information. 

Using the `Cell` prop would be better approach:

https://stackoverflow.com/questions/57877482/react-table-render-component-inside-data

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Thanks @YaiyaSam! I added a comment

File: admin-portal/src/pages/aidPackage/assignSuppliers/assignSuppliers.tsx (hunk 1 above)
Comment (YaiyaSam) [reply admin-portal/src/pages/aidPackage/assignSuppliers/assignSuppliers.tsx]: Hey i changed it. Please review

**Review by jayasanka-sack — APPROVED** [review]
Thanks @YaiyaSam !

**Pull request merged**